snakes_battle/snake.py

```diff
@@ -12,14 +13,20 @@ class Snake:
 
     def __init__(self, color, name) -> None:
 
-        # Picking a random color
         self.name = name
         self.color = color
         
         self.direction = randint(0,3) # Only 4 directions - will pick one of them
         self.length = 1
         self.body_pos = None
 
+        # All the special fruits that active in this snake.
+        self.super_power = {
```

Comment:
Would not use a dictionary here, but simple normal class properties. What do you gain from it being a dict?
The way I see it it's only a room for typo's (because unlike normal properties the key is not auto-completed), and makes the code look worse when using this property.